src/components/StepByStepGuide.tsx

```diff
@@ -0,0 +1,366 @@
+  multi?: boolean;
+  /** Optional custom renderer; defaults to StepPageDiscrete. */
+  component?: React.FC<StepRendererProps>;
+  options?: StepOption[];
+}
+
+interface StepByStepGuideProps {
+  filters: SearchFilters;
+  onFilterChange: <T extends string | number | (string | number)[] | null>(
+    key: keyof SearchFilters,
+    value: T | null,
+  ) => void;
+}
+
+const StepByStepGuide: React.FC<StepByStepGuideProps> = ({
+  filters,
+  onFilterChange,
+}) => {
+  const [isCollapsed, setIsCollapsed] = useState(false);
+  const [currentView, setCurrentView] = useState<"home" | number>("home");
+  // Single source of truth for all facet options
+  const globalFacetOptions = useMemo(
+    () => getGlobalFacetOptions(pathwayMetadata),
+    [],
+  );
+
+  // Normalize provider options -> tile-ready {id,title,value}
+  const normalize = (
+    arr: Array<{ value: string | number; label: string }>,
+  ): StepOption[] =>
+    arr.map((o) => ({ id: String(o.value), title: o.label, value: o.value }));
+
+  // Options by facet for easy access
+  const optionsByFacet: Record<string, StepOption[]> = useMemo(
+    () => ({
+      pathwayType: normalize(globalFacetOptions["pathwayTypeOptions"]),
+      modelTempIncrease: normalize(globalFacetOptions["temperatureOptions"]),
+      geography: normalize(globalFacetOptions["geographyOptions"]),
+      metric: normalize(globalFacetOptions["metricOptions"]),
+      emissionsTrajectory: normalize(
+        globalFacetOptions["emissionsTrajectoryOptions"],
+      ),
+      policyAmbition: normalize(globalFacetOptions["policyAmbitionOptions"]),
+    }),
+    [globalFacetOptions],
+  );
+
+  const descriptions: Record<string, Record<string, string>> = {
+    pathwayType: {
+      "Direct Policy": "Sourced directly from policy frameworks",
+      "Explorative": "Future states under explicit assumptions",
+      "Normative": "Pre-determined target outcome",
+      "Predictive": "Extrapolates from current trends.",
+    },
+    emissionsTrajectory: {
+      "Significant increase": "Grow more than 50% by 2050",
+      "Moderate increase": "Grow 15%-50% by 2050",
+      "Minor increase": "Grow 5% -15% by 2050",
+      "Low or no change": "Change less than 5% by 2050",
+      "Minor decrease": "Decrease 5%-15% by 2050",
+      "Moderate decrease": "Decrease 15%-50% by 2050",
+      "Significant decrease": "Decrease more than 50% by 2050",
+      "": "Include any emissions trajectory",
+    },
+    policyAmbition: {
+      "No policies included": "Excludes policy impacts.",
+      "Current/legislated policies": "Only policies already in place.",
+      "Current and drafted policies": "Includes in-process policy drafts.",
+      "NDCs, unconditional only": "Policies to reach unconditional targets",
+      "NDCs incl. conditional targets": "Policies to reach all targets",
+      "Normative/Optimization-based": "Policies to reach a climate outcome.",
+      "": "Include any policy ambition",
```

Comment:
@jacobvjk Similarly, is "Other policy ambition" a value we want users to be able to select?

Reply:
Do you mean "Any policy ambition" or "Other policy ambition" @AlexAxthelm ? I think we should add both, with "any" again effectively removing the filter